### ISSUE TITLE
This plugin is deprecated and archived

### ISSUE THREAD

**Issue by mrmlnc** [mrmlnc/vscode-stylefmt#164]
The main package (`stylefmt`) that uses this extension is no longer being developed. I don't see the point in using outdated technology.

Instead of `stylefmt` you can use the following solutions:

* [`postcss`](https://github.com/postcss/postcss) with similar plugins.
* [`stylelint`](https://github.com/stylelint/stylelint)
* Maybe something else

I am releasing the last release of this extension and archiving it. 

I am ready to back to the plugin support if the main package find a maintainer and back to the active development. Right now, this is the **last** release (not a **latest**).

**Comment by rollecode** [comment]
Stylefmt is better than prettier and others which force some styling preferences so I will keep using it. [This PR](https://github.com/mrmlnc/vscode-stylefmt/pull/163) with [these updates](https://github.com/mrmlnc/vscode-stylefmt/pull/163/commits/f1669822d943d467df8a0c3146d384cf0b0904ee) fixes this plugin even though the main package is no longer developed. It uses [my fork of stylefmt](https://github.com/ronilaukkarinen/stylefmt) that has [been updated to fixed packages](https://github.com/ronilaukkarinen/stylefmt/commit/9c57a0814fddb475ff2d030b3a364b2182470967).

The funny thing is, practically [stylefmt](https://github.com/morishitter/stylefmt) works perfectly fine as long as [stylelint](https://github.com/stylelint/stylelint) is kept up to date once or twice a year, because there will be some slight changes in stylelint rule sets that will be outdated in some points. Very small things but affect everyone because packages are not updated. See my other related issues: [here](https://github.com/morishitter/stylefmt/issues/334), [here](https://github.com/digitoimistodude/air-light/issues/31), [here](https://github.com/morishitter/stylefmt/issues/334#issuecomment-438234236) and [here](https://github.com/morishitter/gulp-stylefmt/issues/10#issuecomment-436571016).

So basically it means that stylefmt depends on packages like stylelint and it's broken ONLY because the developer is too lazy to run `npm update`... the stylefmt itself is functioning as long as you update the packages. The maintainer has not updated stylelint for years as seen [here](https://github.com/morishitter/stylefmt/blob/ee59bf64c27b15d45b0630ac52338fcae0fcc8ee/package.json#L44), the newest version is 13.6.1. The world we live in...

**Comment by rollecode** [comment]
So for future refrence and other people (you might want to add this to README @mrmlnc if you are not interested in upkeeping this repository any more nor merging my [PR](https://github.com/mrmlnc/vscode-stylefmt/pull/163)) to get vscode-stylefmt working again:

1. `cd ~/.vscode/extensions/mrmlnc.vscode-stylefmt-2.6.1`
2. `code package.json`
3. Edit lines shown [here](https://github.com/mrmlnc/vscode-stylefmt/pull/163/commits/f1669822d943d467df8a0c3146d384cf0b0904ee) and save package.json
4. Run `npm update`
5. Restart VSCode / Reload extensions from disk

Tested and confirmed working with latest and clean vscode-stylefmt.

**Comment by mrmlnc** [comment]
JFYI: I can transfer this repository to you. In this case, you will need to publish the extension in the store under [your publisher](https://github.com/remarkjs/remark/issues/508#issuecomment-653866681). Alternatively, you can add your changes to the root repository (`stylefmt`) or become its maintainer.

**Comment by rollecode** [comment]
I don't really have any other VSCode plugins so the world is unfamiliar to me. If you mean [this stylefmt repository](https://github.com/morishitter/stylefmt), its maintainer morishitter has been in the dark for two years and that's the origin of most issues so that won't help. I use stylefmt daily so would be glad to take over. I just need some time to create my own version of the plugin, you can then point this one to the new one. I'll let you know when I have time, may take a while though.

**Comment by rollecode** [comment]
Hey @mrmlnc, I now have my active fork as a working VSCode plugin:

[Source](https://github.com/ronilaukkarinen/vscode-stylefmt)
[Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=ronilaukkarinen.vscode-stylefmt)

**Comment by rollecode** [comment]
@mrmlnc Do you have an idea why it says

```
Activating extension 'ronilaukkarinen.vscode-stylefmt' failed: Cannot find module 'postcss'
Require stack:
- /Users/rolle/.vscode/extensions/ronilaukkarinen.vscode-stylefmt-2.7.6/out/stylefmt/index.js
- /Users/rolle/.vscode/extensions/ronilaukkarinen.vscode-stylefmt-2.7.6/out/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js.
```

Everything should be all right and I haven't done any modifications other than installed travis and updated dependencies... 🤔  I'm kinda new to this as this is my first vscode plugin.

**Comment by rollecode** [comment]
On development mode <kbd>F5</kbd> it works though... 

```
Loading development extension at /Users/rolle/Projects/vscode-stylefmt
```

Why is that?

**Comment by rollecode** [comment]
...well, I had node_modules inside .vscodeignore... Thank goodness it works now. [(ref)](https://github.com/cake-build/cake-vscode/issues/92#issuecomment-496998078)

**Comment by rollecode** [comment]
@mrmlnc As a reminder, I have active repos for both [vscode-stylefmt](https://github.com/ronilaukkarinen/vscode-stylefmt) and [stylefmt](https://github.com/ronilaukkarinen/stylefmt), you could point your users to this direction. Thanks.

**Comment by mrmlnc** [comment]
A link to your extension has been added to the plugin description.
